AutorunsToWinEventLog/AutorunsToWinEventLog.ps1

```diff
@@ -32,3 +35,43 @@ Foreach ($item in $autorunsArray) {
   $item = $(Write-Output $item  | Out-String -Width 1000)
   Write-EventLog -LogName Autoruns -Source AutorunsToWinEventLog -EntryType Information -EventId 1 -Message $item
 }
+
+# Collect principals in interesting local groups (Administrators, Remote Desktop Users, and DCOM Users)
+# Requires PowerShell 5.1 due to usage of Get-NetLocalGroup and Get-LocalGroupMember
+
+# Get the FQDN of the current user's domain. Todo: update this method to support foreign security principals in local groups.
+$DomainFQDN = $ENV:USERDNSDOMAIN
```

Comment:
Just tested this locally and unfortunately the domain resolution here doesn't return the expected value, most likely because this script ends up being run as SYSTEM by the task scheduler: https://github.com/palantir/windows-event-forwarding/blob/master/AutorunsToWinEventLog/Install.ps1#L18-L21